setup/agreement/odoo/addons/agreement

```diff
@@ -0,0 +1 @@
+../../../../agreement
```

Comment:
This should comes in a separated commit named  [MIG] agreement: pre-commit execution